src/test/java/seedu/address/storage/JsonAdaptedAppointmentTest.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.storage;
```

Comment:
LGTM, need to include tests for JSONAdaptedDate/JSONAdaptedFrom/JSONAdaptedTo

Reply:
Included in the newest commit. Please take a look.